uwblueprint/extend-a-family
# add manage user page

## Notion ticket link

[Ticket Name](https://www.notion.so/uwblueprintexecs/7e51fde5f22d448b8226f2cfa4dd93d5?v=b2dc0dba06574651a8b6063bdd0d74e0&p=6bddda91d13842c3951f993ffbe5eea9&pm=s)



## Implementation description
* Added a page /manage-users where you can see all users based on a certain role
* Made it such that it paginates to overwhelm people when there are lots of users (prolly won't happen)



## Steps to test
1. In an admin account, go to /manage-users
2. By default it should display the admin users in the table
3. Changing the selected role should also change the users displayed
4. There is a footer for the table to adjust number of users per page as well as changing the page. Since I nuked the database's users, theres only so many, so you can try manually changing the number of rows per page and see if it works correctly that way



## What should reviewers focus on?
* Check how it looks when we have more user/data filling out the table
* Check if the aria-labels and design complies with A11Y


## Checklist
